docs/testnet-guardian-runbook.md

```diff
@@ -0,0 +1,151 @@
+|2| Configuration/infra issues when deploying a QA-net | There are some troubleshooting notes in the notion document for the instructions to deploy a network. If that doesn't help then contact the any of the members listed in owners column for help | Blocker | Conner, Ahmad, Nathan|
+| 3| Unstable Testnet* (criteria for an unstable testnet is listed below)| Create an issue describing the observed behavior. The protocol team would need to investigate the problem and provide a hotfix before proceeding further with the release | Blocker | Aneesha |
+| 4| Errors when completing the challenges| Create an issue describing the observed behavior. Might have to exclude the challenge that causes the error if it cannot be patched and tested within the QA-net phase. Let the marketing and communications team know? | Major | Aneesha, Brandon, Christine|
+| 5 | Other errors/suspicious behavior observed when monitoring. These don't destabilize the testnet or affect the challenges | Create an issue describing the behavior | Minor | |
+| 6 | Patch/hotfix | On receiving hotfix(es) on release and master(required for updating apt package) branch, destroy the existing   QA-net, repeat the [build](#build) and [deploy](#deploy) steps. Perform the [sanity checks](#sanity-check) and verify the fix | | |
+| 7 | Hotfix doesn't work or causes other issues | Revert the fix and let the owners for the event type (for which the fix was issued) know |
+
+*Criteria for an unstable testnet:
+
+    1. Network forks 
+    2. Blocks are not being created 
+    3. Blocks/transactions/snark-work not being gossiped
+    4. Transactions not getting included in a block
+    5. Prover failures
+    6. Other persistent crashes halting block production
+    7. New nodes unable to join the network (either stuck in bootstrap or catchup)
+    8. New nodes not syncing to the main chain and causing forks
+
+## Release
+
+On successful completion of the QA-net phase, the reliability team would spin up a testnet for the release. Testnet guardian would perform the [sanity checks](#sanity-check) on the new testnet and update [brew](#update-brew-package) and [apt](#update-apt-package) packages.
+
+##### Update brew package
+To update brew package,
+
+1. Download the homebrew binary package from gcloud at the location specified in the `Copy artifacts to cloud` task in the `build-macOS` CI job. For example `gs://network-debug/382998/build/homebrew-coda.tar.gz`
+2. Upload it to our public s3 bucket at `https://s3-us-west-2.amazonaws.com/packages.o1test.net`
+3. In `coda.rb` in `CodaProtocol/homebrew-coda`,
+        1. update the url with the location of the new tarball
+        2. update the hash. The file containing the hash is uploaded to gcloud as well, for example `gs://network-debug/382998/build/homebrew-coda.tar.gz.sha256`
+        3. increment the revision number
```

Comment:
See my response to the discussion with Avery -- I think we should add a (4) update seed-peers inside the `coda.rb`.